### DISCUSSION TITLE
شناسه پرداخت کننده به پرداخت ملت

### DISCUSSION THREAD

**Discussion by mojtabamansouri** [ali-zahedi/az-iranian-bank-gateways#63] [Show and tell]
سلام به پرداخت ملت شناسه پرداخت کننده میخواد که باید ما بفرستیم سمت بانک..اونو میشه کاریش کرد؟

**Comment by ali-zahedi** [comment]
سلام وقت بخیر.  
دقیق متوجه منظورتون نشدم ولی درگاه به پرداخت تا جایی که من درجریان هستم درست کار میکنه فقط کافیه راهنما رو دنبال کنید و متغیر ها رو در تنظیمات به درستی ست کنید و با ای پی اعلام شده به بانک تست کنید. 

اگه سوالتون چیز دیگه است با جزئیات بیشتر و به صورت مشخص بفرمایید. اگه راهنما یا مستنداتی هم لازم هست لینکش و قرار بدید که بتونیم بررسی کنیم با هم.
